docs/sources/installation/configure-docker.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "Configure Grafana Docker image"
+description = "Guide for configuring the Grafana Docker image"
+keywords = ["grafana", "configuration", "documentation", "docker"]
+type = "docs"
+[menu.docs]
+name = "Configure Grafana Docker image"
+identifier = "docker"
+parent = "administration"
+weight = 650
++++
+
+# Configure a Grafana Docker image
+
+If you are running Grafana in a Docker image, then you configure Grafana using [environment variables]({{< configuration.md#configure-with-environment-variables >}}) rather than directly editing the configuration file. If you want to save your data, then you also need to designate persistent storage or bind mounts for the Grafana container.
```

Comment:
Too fix failing website build:

```suggestion
If you are running Grafana in a Docker image, then you configure Grafana using [environment variables]({{< relref "configuration.md#configure-with-environment-variables" >}}) rather than directly editing the configuration file. If you want to save your data, then you also need to designate persistent storage or bind mounts for the Grafana container.
```